src/containers/projectcontainer.js

```diff
@@ -52,6 +52,21 @@ class ProjectContainer extends Component {
     this.props.updateField(fieldName, value);
   }
 
+  generateProjectId = (customerId) => {
+    const prefix = this.props.customers.data.get(customerId).name
+      .substring(0, 3)
```

Comment:
It would be safer to pick out `/^\D+/` so that other prefix lengths will work too.